### PR TITLE
[TASK] Mention configuration option for fileadmin folder

### PR DESCRIPTION
**Pull request by brotkrueml**
Releases: main, 12.4, 11.5

### CODE REVIEW

File: Documentation/ApiOverview/DirectoryStructure/Index.rst [hunk 1]
```diff
@@ -184,6 +184,10 @@ further storage locations configured outside of :file:`fileadmin/`, even
 pointing to different servers or using 3rd party digital asset management
 systems.
 
+Depending on the configuration in
+:ref:`$GLOBALS['TYPO3_CONF_VARS']['BE']['fileadminDir'] <typo3ConfVars_be_fileadminDir>`
+another folder name than :file:`fileadmin/` can be in use.
```
Comment (linawolf) [inline Documentation/ApiOverview/DirectoryStructure/Index.rst:189]: I think we should also warn that not all extensions might respect this setting?

Comment (brotkrueml) [reply Documentation/ApiOverview/DirectoryStructure/Index.rst:189]: This should then be done in the section of the configuration setting, which is not in the scope of this PR. The note only hints, that a fileadmin folder might not be available.

Comment (brotkrueml) [reply Documentation/ApiOverview/DirectoryStructure/Index.rst:189]: And: If you change it you have to take care about renaming the existing fileadmin folder and change the storage record accordingly.

Comment (linawolf) [reply Documentation/ApiOverview/DirectoryStructure/Index.rst:189]: Ack

**Review by linawolf — APPROVED** [review]

**Pull request merged**